build/stylesheet/lib/parse-component.js

```diff
@@ -0,0 +1,55 @@
+import { parse } from 'react-docgen';
+import annotationResolver from 'react-docgen-annotation-resolver';
+
+import readFile from './read-file';
+
+const replaceValue = val => {
+  const map = {
+    'SIZES.KILO': 'kilo',
+    'SIZES.MEGA': 'mega',
+    'SIZES.GIGA': 'giga',
+    'SIZES.TERA': 'tera',
+    'SIZES.PETA': 'peta',
+    'SIZES.EXA': 'exa',
+    'SIZES.ZETTA': 'zetta',
+    true: true,
+    false: false
+  };
```

Comment:
Do we need to replace all constants?

Reply:
Yes! Ideally programmatically at some point.